mod_reforged/hooks/config/strings.nut

```diff
@@ -252,7 +252,7 @@ local vanillaDescriptions = [
 			Effects = [{
 				Type = ::UPD.EffectType.Passive,
 				Description = [
-					"Reduce the [Hitpoint|Concept.Hitpoints] damage threshold for triggering [Morale Checks|Concept.Morale] with your attacks to 1.",
+					"Reduce the [Hitpoint|Concept.Hitpoints] damage threshold for triggering [morale checks|Concept.Morale] with your attacks to 1.",
```

Comment:
I'd change this line to `The damage threshold for triggering morale checks with your attacks is reduced to 1`